MAINTAINERS

```diff
@@ -1,8 +1,5 @@
-Aaron Lehmann <aaronl@vitelus.com> (@aaronlehmann)
-Brendan Burns <bburns@microsoft.com> (@brendandburns)
 Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
-Jason Bouzane <jbouzane@google.com> (@jbouzane)
-John Starks <jostarks@microsoft.com> (@jstarks)
-Jonathan Boulle <jon@nstack.com> (@jonboulle)
 Stephen Day <stephen.day@docker.com> (@stevvooe)
```

Comment:
@stevvooe Your email address needs to be updated?

Reply:
ooh yes
i'll look up the gmail one